Bridge build error: Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll not found

## Description
Build fails with MSB3106 warning because the DLL name doesn't match.

**Current behavior:**
```
D365MetadataBridge net48 win-x86 succeeded with 2 warning(s) (0.6s) → bin\Release\D365MetadataBridge.exe
    C:\Program Files\dotnet\sdk\10.0.201\Microsoft.Common.CurrentVersion.targets(2451,5): warning MSB3106: Assembly strong name "C:\AosService\PackagesLocalDirectory\bin\Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\).
    C:\Program Files\dotnet\sdk\10.0.201\Microsoft.Common.CurrentVersion.targets(2451,5): warning MSB3106: Assembly strong name "C:\AosService\PackagesLocalDirectory\bin\Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\).
```

**Environment:**
- D365FO Version: 10.0.46 (10.0.2428.63)
- Platform Version: Update70 (7.0.7778.29)
- One-box VM

**Root cause:**
In this D365FO version, the DLL is named `Microsoft.Dynamics.Framework.Tools.MetaModel.Core.17.0.dll` instead of `Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll`.

**Suggested fix:**
Remove the explicit reference to `Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll` from `bridge/D365MetadataBridge/D365MetadataBridge.csproj` since it's already covered by the wildcard pattern `$(D365BinPath)\Microsoft.Dynamics.*.dll`.

--- a/bridge/D365MetadataBridge/D365MetadataBridge.csproj
+++ b/bridge/D365MetadataBridge/D365MetadataBridge.csproj
@@ -13,8 +13,12 @@
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
     <!-- MSB3277: version conflict between Microsoft-shipped SCDP.dll (7.0.0) and
          SCDPBundling.dll (depends on 7.0.4) in PackagesLocalDirectory\bin.
-         We don't control these DLLs and the bridge doesn't use SCDP at all — safe to suppress. -->
-    <MSBuildWarningsAsMessages>MSB3277</MSBuildWarningsAsMessages>
+         We don't control these DLLs and the bridge doesn't use SCDP at all — safe to suppress.
+         MSB3106: some D365FO versions ship MetaModel.Core with a version suffix in the filename
+         (e.g. Microsoft.Dynamics.Framework.Tools.MetaModel.Core.17.0.dll); the wildcard glob
+         picks it up automatically, so the explicit name reference is gone and this warning should
+         no longer appear — kept here as a safety net for edge-case environments. -->
+    <MSBuildWarningsAsMessages>MSB3277;MSB3106</MSBuildWarningsAsMessages>
   </PropertyGroup>
 
   <!--
@@ -53,9 +57,6 @@
   </ItemGroup>
   <ItemGroup Condition="'$(D365BinPath)' != ''">
     <Reference Include="$(D365BinPath)\Microsoft.Dynamics.*.dll">
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="$(D365BinPath)\Microsoft.Dynamics.Framework.Tools.MetaModel.Core.dll">
       <Private>false</Private>
     </Reference>
   </ItemGroup>

